plenum/test/node_catchup/test_catchup_inlcuding_3PC.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from plenum.common.constants import DOMAIN_LEDGER_ID
+from plenum.common.util import check_if_all_equal_in_list
+from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
+from plenum.test.node_catchup.helper import check_last_3pc_master, \
+    waitNodeDataEquality
+from plenum.test.pool_transactions.helper import buildPoolClientAndWallet
+from plenum.test.test_client import TestClient
+from stp_core.loop.eventually import eventually
+
+
+def chk_if_equal_txn_to_3pc(nodes, count=None):
```

Comment:
No, since `txn_seq_range_to_3phase_key` is only filled when nodes order requests and not for data received from catchup